build-config/src/main/resources/forbidden-common.txt

```diff
@@ -0,0 +1,115 @@
+# Hibernate Validator, declare and validate application constraints
+#
+# License: Apache License, Version 2.0
+#  See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+
+# This file contains forbidden API signatures common to all modules
+
+@ignoreUnresolvable
+@defaultMessage Shouldn't be used
+
+################################################################################################################
+# Some packages that should be used
+java.awt.**
+sun.**
+org.slf4j.**
+junit.framework.**
+
+################################################################################################################
+# org.testng.Assert#fail() should be used
+org.assertj.core.api.Fail#fail()
+org.testng.FileAssert#fail()
+org.assertj.core.api.Assertions#fail()
+
+################################################################################################################
+# Nobody should be using StringBuffer anymore
+java.lang.StringBuffer
```

Comment:
Not sure about that ... I'll check if it'll work with the class only, and let you know
There's not much about the format of these files (or in other words - I haven't found anything about the format in the documentation of the plugin :) so I just looked at format of provided files, but I haven't checked for such cases ...)